pkg/cluster/kubernetes/sync.go

```diff
@@ -145,6 +145,11 @@ func (c *Cluster) collectGarbage(
 
 		switch {
 		case !ok: // was not recorded as having been staged for application
+			if res.Policies().Has(policy.DisableGarbageCollect) {
+				c.logger.Log("info", "not deleting resource; garbage colelction annotation in file", "resource", res.ResourceID())
```

Comment:
```suggestion
				c.logger.Log("info", "not deleting resource; garbage collection disabled by annotation", "resource", res.ResourceID())
```

(typo, and the annotation is in the cluster resource, not the file)